console/console-init/ui/src/components/JsonEditor/JsonEditor.tsx

```diff
@@ -16,12 +16,14 @@ import { IAceEditorProps } from "react-ace";
 
 interface IJsonEditorProps extends IAceEditorProps {
   setDetail?: (detail: string | undefined) => void;
+  manageState?: boolean;
```

Comment:
Please include a meaningful name here.

Reply:
What state is being managed? Is it to make enable or disable json editor? Please choose the name accordingly.